dev-docs/show-native-ads.md

```diff
@@ -166,19 +166,35 @@ And the following optional fields:
 + icon
 + cta
 
-A native `image` ad unit can be set up in the manner below:
+A native "image-type" ad unit can be set up as shown in the following example.
```

Comment:
See note above about documenting "mediaTypes.native.image.sizes" and "mediaTypes.native.image.aspect_ratios"